k8s/charts/seaweedfs/templates/admin/admin-servicemonitor.yaml

```
@@ -0,0 +1,34 @@
+  endpoints:
+    - interval: 30s
+      port: metrics
+      scrapeTimeout: 5s
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "seaweedfs.name" . }}
+      app.kubernetes.io/component: admin
+{{- end }}
+{{- end }}
+{{- end }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)